components/Quiz.tsx

```diff
@@ -0,0 +1,192 @@
+/* eslint-disable max-len */
+
+import CorrectIcon from './icons/Correct';
+import IncorrectIcon from './icons/Incorrect';
+import { useState } from 'react';
+
+interface IconWrapperProps {
+  children: React.ReactNode
+}
+export enum QuizType {
+  trueOrFalse = '是非題',
+  quickReview = '快速回顧'
+}
+interface QuizProps {
+  type: QuizType,
+  options: QuizOption[],
+  children: React.ReactNode
+}
+interface QuizHeadingProps {
+  type: QuizType,
+  children: React.ReactNode
+}
+interface QuizOptionProps {
+  option: QuizOption,
+  answer: QuizOption,
+  isSubmit: boolean,
+  isCorrect: boolean,
+  onClick: Function
+}
+interface QuizOption {
+  text: string,
+  isCorrect: boolean
+}
+interface QuizSubmitButtonProps {
+  onClick: Function
+}
+interface QuizAnswerReult {
+  isCorrect: boolean,
+  answer: QuizOption,
+  options: QuizOption[]
+}
```

Comment:
```suggestion
export enum QuizType {
  trueOrFalse = '是非題',
  quickReview = '快速回顧'
}

interface QuizProps {
  type: QuizType,
  options: QuizOption[],
  children: React.ReactNode
}

interface QuizHeadingProps {
  type: QuizType,
  children: React.ReactNode
}

interface QuizOptionProps {
  option: QuizOption,
  answer: QuizOption,
  isSubmit: boolean,
  isCorrect: boolean,
  onClick: Function
}

interface QuizOption {
  text: string,
  isCorrect: boolean
}

interface QuizSubmitButtonProps {
  onClick: Function
}

interface QuizAnswerReult {
  isCorrect: boolean,
  answer: QuizOption,
  options: QuizOption[]
}
```

It would be better to have newlines between interface declarations.